man/systemd.exec.xml

```diff
@@ -1868,8 +1868,9 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         memory segments as executable are prohibited.  Specifically, a system call filter is added that rejects
         <citerefentry><refentrytitle>mmap</refentrytitle><manvolnum>2</manvolnum></citerefentry> system calls with both
         <constant>PROT_EXEC</constant> and <constant>PROT_WRITE</constant> set,
-        <citerefentry><refentrytitle>mprotect</refentrytitle><manvolnum>2</manvolnum></citerefentry> system calls with
-        <constant>PROT_EXEC</constant> set and
+        <citerefentry><refentrytitle>mprotect</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        or <citerefentry><refentrytitle>pkey_mprotect</refentrytitle><manvolnum>2</manvolnum></citerefentry>
```

Comment:
Since https://github.com/systemd/systemd/pull/6952/commits/ff217dc3afe95504e48aeb8d8ad7fb7f53ce9cb1 was merged, `systemd` has started ignoring system calls unknown to the version of `libseccomp` installed, so it would possibly useful to mention that the filters might not work when `libseccomp` is too old.